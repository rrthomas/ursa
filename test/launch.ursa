let f = fn(n) {
  for i of range(n) {
    print(n, i)
  }
}

launch f(10)
launch f(5)
