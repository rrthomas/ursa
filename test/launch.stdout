10 0
5 0
10 1
5 1
10 2
5 2
10 3
5 3
10 4
5 4
10 5